README.md

```diff
@@ -0,0 +1,80 @@
+# Nexmo JWT Generator for Ruby
+
+[![Gem Version](https://badge.fury.io/rb/nexmo-jwt.svg)](https://badge.fury.io/rb/nexmo-jwt) [![Build Status](https://api.travis-ci.org/Nexmo/nexmo-jwt-ruby.svg?branch=master)](https://travis-ci.org/Nexmo/nexmo-jwt-ruby) [![Coverage Status](https://coveralls.io/repos/github/Nexmo/nexmo-jwt-ruby/badge.svg?branch=coveralls)](https://coveralls.io/github/Nexmo/nexmo-jwt-ruby?branch=master)[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.0%20adopted-ff69b4.svg)](CODE_OF_CONDUCT.md)
+
+<img src="https://developer.nexmo.com/assets/images/Vonage_Nexmo.svg" height="48px" alt="Nexmo is now known as Vonage" />
+
+This is the Ruby library to generate Nexmo JSON Web Tokens (JWTs). To use it you'll
+need a Nexmo account. Sign up [for free at nexmo.com][signup].
+
+* [Requirements](#requirements)
+* [Installation](#installation)
+* [Usage](#usage)
+* [Documentation](#documentation)
+* [License](#license)
+
+## Requirements
+
+The JWT generator supports Ruby version 2.5 or newer.
+
+## Installation
+
+To install the Ruby client library using Rubygems:
+
+    gem install nexmo-jwt
+
+Alternatively you can clone the repository:
+
+    git clone git@github.com:Nexmo/nexmo-jwt-ruby.git
+
+## Usage
+
+By default the Nexmo JWT generator creates a short lived JWT (15 minutes) per request.
+To generate a long lived JWT for multiple requests, specify a longer value in the `exp`
+parameter during initialization. 
+
+Example with no custom configuration:
+
+```ruby
+@builder = Nexmo::JWTBuilder.new(application_id: YOUR_APPLICATION_ID, private_key: YOUR_PRIVATE_KEY)
+@token = @builder.jwt.generate
+```
+
+Example providing custom configuration options:
+
+```ruby
+@builder = Nexmo::JWTBuilder.new(
```

Comment:
I could go either way on this, so I'll leave it up to you.

I liked in the original version how you could pass a block e.g.:

```
Nexmo::JWTBuilder.build(application_id = YOUR_APPLICATION_ID, private_key = YOUR_PRIVATE_KEY) do |builder|
  builder.ttl = 500
  builder.addPath("/messages", {}) # etc
end
```

Without using the block, how do we enable methods like `addPath` for dynamic permissions?

Reply:
This revised version is a more classically defined Ruby form of instantiation.